src/Util/ClassSourceManipulator.php

```diff
@@ -270,10 +270,27 @@ public function addAccessorMethod(string $propertyName, string $methodName, $ret
 
     public function addGetter(string $propertyName, $returnType, bool $isReturnTypeNullable, array $commentLines = []): void
     {
-        $methodName = ('bool' === $returnType ? 'is' : 'get').Str::asCamelCase($propertyName);
+        $methodName = $this->getGetterName($propertyName, $returnType);
         $this->addCustomGetter($propertyName, $methodName, $returnType, $isReturnTypeNullable, $commentLines);
     }
 
+    /**
+     * @return string|null : getter prefix or null if no prefix to append
+     */
+    private function getGetterName($propertyName, $returnType): ?string
```

Comment:
`$propertyName` should by typed with `string`. And as @nicolas-grekas suggested, the return type should not be `null`able

`$returnType` can be left without a type - adding a type is out of scope in this PR (requires changes elsewhere)